pipelines/upload_transacao_cct/tasks.py

```
@@ -291,3 +302,71 @@ def save_upload_timestamp_redis(
+    dates: list[str],
+    full_refresh: bool,
+):
+    where = "true" if full_refresh else ", ".join(dates)
+    sql = f"""
+        SELECT
+            *,
+            NOW() AS datetime_extracao_teste
+        FROM
+            public.{constants.TRANSACAO_POSTGRES_TABLE_NAME.value}
+        WHERE
+            data IN ({where})
```

Ruff (0.14.1)

Possible SQL injection vector through string-based query construction

(S608)